tests/client_tests.ts

```diff
@@ -161,113 +157,83 @@ describe.each([
         expect(response).toBe(true)
       })
     })
+    test(`${permission} key: get version`, async () => {
+      await client.version().then((response: Types.Version) => {
+        expect(response).toHaveProperty('commitSha', expect.any(String))
+        expect(response).toHaveProperty('buildDate', expect.any(String))
+        expect(response).toHaveProperty('pkgVersion', expect.any(String))
+      })
+    })
+    test(`${permission} key: get system info`, async () => {
+      await client.sysInfo().then((response: Types.SysInfo) => {
+        expect(response).toHaveProperty('memoryUsage', null)
+        expect(response).toHaveProperty('processorUsage', expect.any(Array))
+        expect(response.global).toHaveProperty(
+          'totalMemory',
+          expect.any(Number)
+        )
+        expect(response.global).toHaveProperty('usedMemory', expect.any(Number))
+        expect(response.global).toHaveProperty('totalSwap', expect.any(Number))
+        expect(response.global).toHaveProperty('usedSwap', expect.any(Number))
+        expect(response.global).toHaveProperty('inputData', expect.any(Number))
+        expect(response.global).toHaveProperty('outputData', expect.any(Number))
+        expect(response.process).toHaveProperty('memory', expect.any(Number))
+        expect(response.process).toHaveProperty('cpu', expect.any(Number))
+      })
+    })
+    test(`${permission} key: get pretty system info`, async () => {
+      await client.prettySysInfo().then((response: Types.SysInfoPretty) => {
+        expect(response).toHaveProperty('memoryUsage', expect.any(String))
+        expect(response).toHaveProperty('processorUsage', expect.any(Array))
+        expect(response.global).toHaveProperty(
+          'totalMemory',
+          expect.any(String)
+        )
+        expect(response.global).toHaveProperty('usedMemory', expect.any(String))
+        expect(response.global).toHaveProperty('totalSwap', expect.any(String))
+        expect(response.global).toHaveProperty('usedSwap', expect.any(String))
+        expect(response.global).toHaveProperty('inputData', expect.any(String))
+        expect(response.global).toHaveProperty('outputData', expect.any(String))
+        expect(response.process).toHaveProperty('memory', expect.any(String))
+        expect(response.process).toHaveProperty('cpu', expect.any(String))
+      })
+    })
+    test(`${permission} key: get /stats information`, async () => {
+      await client.stats().then((response: Types.Stats) => {
+        expect(response).toHaveProperty('databaseSize', expect.any(Number))
+        expect(response).toHaveProperty('lastUpdate') // TODO: Could be null, find out why
+        expect(response).toHaveProperty('indexes', expect.any(Object))
+      })
+    })
   })
 })
 
 describe.each([{ client: masterClient, permission: 'Master' }])(
   'Test on routes where only master key has access',
   ({ client, permission }) => {
-    describe('Test on base routes', () => {
```

Comment:
Why is it deleted? (Maybe I see the diff badly because of github...)

Reply:
Because at first those routes were supposed to work for both private key and master key. But, it wasn't. So i had to do two separate instanced of tests only on master key (the one deleted) and only on private key (the one in comment). Since this has been fixed in this version I have added the tests in the main instance of tests were tests are done on both private and master key.